packages/core/src/App/Containers/Redirect/redirect.jsx

```diff
@@ -215,6 +224,8 @@ export default withRouter(
         verification_code: client.verification_code,
         fetchResidenceList: client.fetchResidenceList,
         hasAnyRealAccount: client.hasAnyRealAccount,
+        is_logged_in: client.is_logged_in,
+        is_logging_in: client.is_logging_in,
         openRealAccountSignup: ui.openRealAccountSignup,
         setResetTradingPasswordModalOpen: ui.setResetTradingPasswordModalOpen,
         toggleAccountSignupModal: ui.toggleAccountSignupModal,
```

Comment:
Is it possible for you to refactor this to use observer and useStore instead of this legacy connect method ? 👀

Reply:
Updated.

Reply:
perfecto 🚀